sdcflows/interfaces/fmap.py

```diff
@@ -666,17 +666,21 @@ def au2rads(in_file, newpath=None):
     data = im.get_fdata(dtype='float32')
     hdr = im.header.copy()
 
-    # First center data around 0.0.
-    data -= mode(data, axis=None)[0][0]
+    if np.allclose((data.min(), data.max()), (-np.pi, np.pi), atol=0.01):
+        # Already in rads, but wrap to 0 - 2pi
+        data[data < 0] += 2 * np.pi
+    else:
+        # First center data around 0.0.
+        data -= mode(data, axis=None)[0][0]
 
-    # Scale lower tail
-    data[data < 0] = - np.pi * data[data < 0] / data[data < 0].min()
+        # Scale lower tail
+        data[data < 0] = - np.pi * data[data < 0] / data[data < 0].min()
 
-    # Scale upper tail
-    data[data > 0] = np.pi * data[data > 0] / data[data > 0].max()
+        # Scale upper tail
+        data[data > 0] = np.pi * data[data > 0] / data[data > 0].max()
```

Comment:
This (the old code) looks problematic - we shift the left side of the pyramid first, hence making it overlap with the right hand side. Operations should be done the other way around.